src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/helper/LetterAddressHelper.java

```diff
@@ -1,18 +1,18 @@
 package uk.gov.hmcts.reform.finrem.caseorchestration.helper;
 
-import org.springframework.stereotype.Component;
-
 import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.List;
 import java.util.Map;
 
-@Component
 public class LetterAddressHelper {
 
-    private List<String> addressLines = new ArrayList<>();
+    private LetterAddressHelper() {
```

Comment:
Think we've agreed to use annotations here https://github.com/hmcts/finrem-case-orchestration-service/pull/278/files to prevent this